src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -863,7 +863,7 @@ private function loadDeleteForm(): void
         $this->template->assign('deleteForm', $deleteForm->createView());
     }
 
-    private function getHiddenJsonField(string $name, string $json): SpoonFormHidden
+    private function getHiddenJsonField(string $name, ?string $json = null): SpoonFormHidden
```

Comment:
you said that you always need to pass 2 parameters, and that doesn't match your current implementation